remote-inference.md

```diff
@@ -0,0 +1,79 @@
+# Inferencing with the fine-tuned model
+After the adapters are trained in the remote environment, you can use a simple Gradio application to interact with the model.  
+
+![Fine-tune complete](Images/remote/finetune-res.png)
+
+## Provision Azure Resources
+Similar to the fine-tuning process, you need to set up the Azure Resources for remote inference by executing the `AI Toolkit: Provision Azure Container Apps for inference.` from the command palette. During this setup, you will be asked to select your Azure Subscription and resource group.  
+![Provision Inference Resource](Images/remote/provision-inference.png)
+   
+By default, the subscription and the resource group for inference should match those used for fine-tuning. The inference will use the same Azure Container App Environment and access the model and model adapter stored in Azure Files, which were generated during the fine-tuning step. 
+
+Once provisioning is successfully completed, you can find the web API endpoint under `ACA_APP_ENDPOINT` within `./infra/inference.config.json`. You are now ready to evaluate the model using this endpoint.
```

Comment:
No need to run deploy command?

Reply:
Renamed `Update code for inference` to be `Deploy for inference`